dshr/dshr_mod.F90

```diff
@@ -309,18 +312,18 @@ subroutine dshr_mesh_init(gcomp, nullstr, logunit, compname, model_nxg, model_ny
                 write(logunit,*) ' single column mode, lon lat = ',scol_mode, scol_lon, scol_lat
              end if
           else
-             scol_lon  = 1.e36_r8
-             scol_lat  = 1.e36_r8
+             scol_lon  = shr_const_spval
+             scol_lat  = shr_const_spval
           end if
        else
           scol_mode = .false.
-          scol_lon  = 1.e36_r8
-          scol_lat  = 1.e36_r8
+          scol_lon  = shr_const_spval
+          scol_lat  = shr_const_spval
        end if
 
        ! Now create the model meshfile using the model_maskfile as the input file
        call dshr_mesh_create(trim(model_createmesh_fromfile), scol_mode, scol_lon, scol_lat, &
-            trim(compname), my_task, master_task, logunit, model_mesh, model_mask, model_frac, rc=rc)
+            trim(compname), my_task, logunit, model_mesh, model_mask, model_frac, rc=rc)
```

Comment:
Shouldn't this be masterproc and not my_task?

Reply:
No. The logic I am consistently using is that master_task is a module variable set to 0. 
my_task is passed to the various subroutines and then the check is made if my_task == master_task.
I don't think its a good idea to use masterproc in some places and my_task == master_task in others.
I think this is consistently used in dshr_mod.

Reply:
It wasn't consistently done - see my note above.